articles/azure-web-pubsub/reference-cloud-events.md

```diff
@@ -131,7 +131,7 @@ ce-connectionState: eyJrZXkiOiJhIn0=
 
 * `subprotocols`
 
-    The `connect` event forwards the subprotocol and authentication information to Upstream from the client. The Azure SignalR Service uses the status code to determine if the request will be upgraded to WebSocket protocol.
+    The `connect` event forwards the subprotocol and authentication information to Upstream from the client. Web PubSub Service uses the status code to determine if the request will be upgraded to WebSocket protocol.
```

Comment:
```suggestion
    The `connect` event forwards the subprotocol and authentication information to Upstream from the client. Web PubSub service uses the status code to determine if the request will be upgraded to WebSocket protocol.
```
Fixing capitalization per [the documentation](https://learn.microsoft.com/en-us/azure/azure-web-pubsub/)